lib/opal/nodes/top.rb

```diff
@@ -108,6 +172,66 @@ def compile_end_construct
       def version_comment
         "/* Generated by Opal #{Opal::VERSION} */"
       end
+
+
+      def self.module_name(filenamepath, original_filepath = nil, original_filename = nil)
+        original_filename = filenamepath unless original_filename
+        original_filepath = Pathname.new(filenamepath).expand_path unless original_filepath
+        o_s = original_filepath.to_s
+        path, _ = Pathname.new(filenamepath).expand_path.split
```

Comment:
Please, add an argument to the `split` method call. For example I don't remember what's the default value (and I think I'm not the only one)

Reply:
Thats Pathname#split and doesn't accept arguments. I added comments in the code.